README.md

```diff
@@ -1 +1,52 @@
+* All board repositories should be named Team-BoardName (ex. BPS-LeaderPCB) - all board names should end in PCB
+* All board repositories should have zip file of gerbers for the board
+* All board repositories should contain pdf with images of board schematic and board layout (with dimensions)
+* All board repositories should have README that contains the following information
+    * Boards Purpose
+    * Mouser link BOM for Board
+    * List of components that take up vertical/horizontal volume outside of regular board dimensions (for mechanical purposes)
+
+# Pull Request Template
+The file PULL_REQUEST_TEMPLATE.md should be copied into the main board repository directory. This file creates a checklist of 
+things the hardware engineer must complete after opening a PR. After submitting a PR, you must ensure this checklist is completed. Only after
+the checklist is completed will a reviewer review your board.
+
+    **NOTE**: After making changes to your board, it is essential that you re-request a review from the reviewers. Otherwise they will not know when
+to review your design again. YOU MUST BE PROACTIVE IN YOUR WORK!
+
+# Adding Components to library
+1. Add component information to Standardized Mouser Components Excel Sheet (this is so other designs can take this component into consideration if the use is applicable).
+    * Be sure to include component name, mouser cart link, and footprint.
+2. Open the PCB that you are working on in KiCAD (under its repository)
+3. Click on `Symbol Editor`, select a `UTSVT` library to add it to, and click on `Add a New Symbol`.
+4. Download component symbol and footprint (or make one yourself using symbol/footprint editor in KiCAD)
+    * You may not need to download footprints/symbols for some components. Some footprints/symbols will work for multiple components if they are same package/pinout.
+5. If component is large (vertically or horizontally), download 3d model for component.
+6. All symbols should have the following information filled out when added:
+    * Name
+    * Datasheet/Mouser link
+    * Footprint Association
+7. If adding footprint, click on `Footprint Editor`, select a `UTSVT` folder to add it to.
+8. If component takes up vertical or horizontal space (buck converters, connectors, large capacitors, etc.), click on `Edit Foootprint Properties` -> `3D Models` and add 3d model association with footprint.
+9. After filling out the above information, enter the submodule directory through your PCB repository.
+10. There will be changes made to this submodule if everything was done correctly. Now you must commit your changes to a new branch and open a Pull Request.
+11. After the Pull Request is merged, update the submodule in your PCB repository through `git submodule update`.
+
+# Working on a Board
+* The first page of the KiCAD schematic should have a version control description of the board that is being worked on. It should list the changes between each version of the board.
```

Comment:
I approve of this :)

Reply:
got it from mppt schematic, mad beautiful